docs/api/send_receipt.md

```diff
@@ -1,16 +1,17 @@
-Requests to this API should only be made when a temporary value is used as the *x_pos_transaction_ref* in calls to *ProcessAuthorisation*. Oxipay will store the specified *x_pos_transaction_ref*, which it will use for transaction reconciliation in any future requests to *ProcessSalesAdjustment*.
+Requests to this API should only be made when a temporary value is used as the *x_pos_transaction_ref* in calls to *ProcessAuthorisation*. Oxipay will store the specified *x_receipt_number*, which it will use for transaction reconciliation in any future requests to *ProcessSalesAdjustment*.
 
 **Method:** *SendReceipt*
 
 <h3>Request</h3>
 
 Parameter &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;| Type | Description
 -----------|------|-------------
-x_pos_transaction_ref | Unicode string | This must be the same reference that would get passed through on future *ProcessSalesAdjustment* requests.
+x_pos_transaction_ref | Unicode string | This must be the same reference used in *ProcessAuthorisation* request.
 x_merchant_id | Unicode string | Merchant identifier as defined by Oxipay
 x_device_id | Unicode string | Unique device identifier for the POS terminal
 x_operator_id | Unicode string | ID of POS/terminal operator
 x_firmware_version | Unicode string | current firmware version of POS device
+x_receipt_number | Unicode string | This must be the same reference that would get passed through on future *ProcessSalesAdjustment* requests.
```

Comment:
This must be the same  as the x_pos_transaction_ref that would get passed through on future *ProcessSalesAdjustment* requests.